error: narrowing conversion of '65000' from 'long int' to 'int'

When trying to compile using platformIO and lgt8f328p I get the following error:
`.pio/libdeps/program_via_ArduinoISP/QMC5883LCompass/src/QMC5883LCompass.cpp:152:80: error: narrowing conversion of '65000' from 'long int' to 'int' inside { } [-Wnarrowing]`

I changed line 152 of `QMC5883LCompass.cpp` from
`int calibrationData[3][2] = {{65000, -65000}, {65000, -65000}, {65000, -65000}};`
to
`int calibrationData[3][2] = {{32767, -32768}, {32767, -32768}, {32767, -32768}};`

and that solved the error and seemed to give the correct results in my test code. I haven't studied the code so I don't know if this is a viable solution.

--- a/changelog.md
+++ b/changelog.md
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v1.2.2]
+### Fixed
+- Issue #25. A previous change to the calibrate() set values out of range from int variable type. Changed to long.
+- Typo in author credit of QMC5883LCompass.cpp.
+
 ## [v1.2.1]
 ### Fixed
 - Bug in getBearing function where negative values would cause incorrect bearings.

--- a/src/QMC5883LCompass.cpp
+++ b/src/QMC5883LCompass.cpp
@@ -149,10 +149,10 @@ void QMC5883LCompass::setSmoothing(byte steps, bool adv){
 
 void QMC5883LCompass::calibrate() {
 	clearCalibration();
-	int calibrationData[3][2] = {{65000, -65000}, {65000, -65000}, {65000, -65000}};
-  	int	x = calibrationData[0][0] = calibrationData[0][1] = getX();
-  	int	y = calibrationData[1][0] = calibrationData[1][1] = getY();
-  	int	z = calibrationData[2][0] = calibrationData[2][1] = getZ();
+	long calibrationData[3][2] = {{65000, -65000}, {65000, -65000}, {65000, -65000}};
+  	long	x = calibrationData[0][0] = calibrationData[0][1] = getX();
+  	long	y = calibrationData[1][0] = calibrationData[1][1] = getY();
+  	long	z = calibrationData[2][0] = calibrationData[2][1] = getZ();
 
 	unsigned long startTime = millis();
 

--- a/src/QMC5883LCompass.cpp
+++ b/src/QMC5883LCompass.cpp
@@ -16,7 +16,7 @@ Supports:
 ===============================================================================================================
 
 v1.0 - June 13, 2019
-Written by MRPrograms 
+Written by MPrograms 
 Github: [https://github.com/mprograms/]
 
 Release under the GNU General Public License v3

